treeherder/webapp/api/performance_data.py

```diff
@@ -267,24 +274,78 @@ class AlertSummaryPagination(pagination.PageNumberPagination):
     page_size = 10
 
 
+class PerformanceAlertSummaryFilter(django_filters.FilterSet):
```

Comment:
This is a little piece of backend mastery. 
It is beyond my knowledge and I would suggest to add a more Django-experienced developer to ensure that the `PerformanceAlertSummaryFilter` is OK.

Reply:
@ionutgoldan it would be a good idea to see also the compiled version of these queries, just to be have a good picture of how it is transformed